test/index.html

```diff
@@ -22,8 +22,8 @@
   <script>
 
     WCT.loadSuites([
-      'basic.html',
-      'basic.html?dom=shadow'
+      'basic.html?dom=shadow',
```

Comment:
`?dom=shadow` tests native SD with Polymer 1, and `wc-shadydom=true&wc-ce=true` tests polyfills with Polymer 2.